app/pb_kits/playbook/packs/examples.js

```diff
@@ -173,3 +173,5 @@ WebpackerReact.setup({
   ...WeekdayStacked,
 })
 
+import * as ListCheckbox from 'pb_list_checkbox/docs'
+WebpackerReact.setup(ListCheckbox)
```

Comment:
Move this to be imported and set the same way all the other kits are getting imported and setup.